Owin.Security.Providers/Gitter/GitterAuthenticationOptions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Owin;
+using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.DataHandler;
+using Owin.Security.Providers.Gitter.Provider;
+
+namespace Owin.Security.Providers.Gitter
+{
+    public class GitterAuthenticationOptions : AuthenticationOptions
+    {
+        /// <summary>
+        ///     Initializes a new <see cref="GitterAuthenticationOptions" />
+        /// </summary>
+        public GitterAuthenticationOptions()
+            : base("Gitter")
+        {
+            Caption = Constants.DefaultAuthenticationType;
+            AuthenticationMode = AuthenticationMode.Passive;
+            CallbackPath = new PathString("/signin-gitter");
+            BackchannelTimeout = TimeSpan.FromSeconds(60);
+            Scope = new List<string>();
+        }
+
+        /// <summary>
+        ///     Get or sets the text that the user can display on a sign in user interface.
+        /// </summary>
+        public string Caption
+        {
+            get { return Description.Caption; }
+            set { Description.Caption = value; }
+        }
+
+        /// <summary>
+        ///     Determines the permissions granted from the application to Gitter.
+        ///     Options include: identify, read, post, client, and/or admin
+        /// </summary>
+        public IList<string> Scope { get; set; }
```

Comment:
Can you please make the setter for Scope private to be in line with all the other ones? This is also to be in line with the ASP.NET 5 providers.